runtime/src/main/java/org/corfudb/protocols/wireprotocol/LogDataSerializerUtils.java

```diff
@@ -0,0 +1,197 @@
+package org.corfudb.protocols.wireprotocol;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.reflect.TypeToken;
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.Unpooled;
+import org.corfudb.common.compression.Codec;
+import org.corfudb.protocols.logprotocol.CheckpointEntry;
+
+import java.util.EnumMap;
+import java.util.Map;
+import java.util.UUID;
+import java.util.concurrent.ConcurrentHashMap;
+
+/**
+ * Utility class used by LogData to help serialize its contents.
+ */
+public class LogDataSerializerUtils {
```

Comment:
Note: since `ICorfuPayload` is being deprecated, this class provides the helper methods necessary to serialize and serialize `LogData` components.